lib/inputstreamhelper.py

```diff
@@ -227,7 +227,13 @@ def _run_cmd(self, cmd, sudo=False):
 
     def _check_loop(self):
         """Check if loop module needs to be loaded into system."""
-        if not self._run_cmd(['modinfo', 'loop'])['success']:
+        loopcheck = False
+        try:
+            loopcheck = self._run_cmd(['modinfo', 'loop'])['success']
+        except:
```

Comment:
We cannot accept bare exceptions, so we need to catch the exact Exception that is thrown here.